src/server/pages/home-page.jsx

```diff
@@ -44,6 +48,26 @@ export default class HomePage extends React.Component {
     const { notebookList } = this.props;
     return (
       <div>
+        <FileDrop
```

Comment:
my `FileDrop` component (for now) targets _an entire page_, not a particular component, so I've added it at the top level here. As designed it can actually go _anywhere_ in a component. Feel free to change that!